### PR TITLE
Feat/2006 custom empty component text

### PR DESCRIPTION
**Pull request by adamhaeger**
## Description

Allows you to configure the empy field text in summary 2.

## Related Issue(s)

- closes [2005](https://github.com/Altinn/app-frontend-react/issues/2005)

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
- Automated tests
  - [x] Unit test(s) have been added/updated
  - [ ] Cypress E2E test(s) have been added/updated
  - [ ] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [ ] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [ ] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [ ] No functionality has been changed/added, so no documentation is needed
  - [x] I will do that later/have created an issue
  
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [ ] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [x] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [x] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

File: src/layout/MultipleSelect/MultipleSelectSummary.tsx [hunk 1]
```diff
@@ -24,11 +28,14 @@ export function MultipleSelectSummary({
     overrides?.displayType === 'list' || (!overrides?.displayType && displayData?.length >= maxStringLength);
   const title = useNodeItem(componentNode, (i) => i.textResourceBindings?.title);
 
+  console.log('emptyFieldText', emptyFieldText);
```
Comment (Magnusrm) [inline src/layout/MultipleSelect/MultipleSelectSummary.tsx:31]: assuming this is from debugging? 😁

**Review by Magnusrm — APPROVED** [review]
Looks good! 👍 Just remove console.log

File: src/layout/MultipleSelect/MultipleSelectSummary.tsx (hunk 1 above)
Comment (Magnusrm) [inline src/layout/MultipleSelect/MultipleSelectSummary.tsx:31]:
```suggestion
```

File: src/layout/MultipleSelect/MultipleSelectSummary.tsx [hunk 2]
```diff
@@ -24,11 +28,14 @@ export function MultipleSelectSummary({
...
+
```
Comment (Magnusrm) [inline src/layout/MultipleSelect/MultipleSelectSummary.tsx:31]:
```suggestion
```

File: src/layout/Summary2/SummaryComponent2/SummaryComponent2.test.tsx [hunk 3]
```diff
@@ -207,6 +207,239 @@ describe('SummaryComponent', () => {
...
+            componentId: 'DropdownId',
+            emptyFieldText,
+          },
+        ],
+      },
+    });
+    expect(container).toHaveTextContent(emptyFieldText);
+  });
+
+  test.only('MultipleSelect: Should render custom empty field text if set in overrides', async () => {
```
Comment (bjosttveit) [inline src/layout/Summary2/SummaryComponent2/SummaryComponent2.test.tsx:385]: Woopsie, was this intended? 😅

Comment (Magnusrm) [reply src/layout/Summary2/SummaryComponent2/SummaryComponent2.test.tsx:385]: 😱

**Pull request merged**